package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "twentynineteen",
+  "version": "1.0.0",
+  "description": "Default WP Theme",
+  "bugs": {
+    "url": "https://github.com/WordPress/twentynineteen/issues"
+  },
+  "homepage": "https://github.com/WordPress/twentynineteen#readme",
+  "devDependencies": {
+    "@wordpress/browserslist-config": "^2.2.2",
+    "autoprefixer": "^9.1.5",
+    "chokidar-cli": "^1.2.1",
+    "node-sass": "^4.9.3",
+    "postcss-cli": "^6.0.0",
+    "rtlcss": "^2.4.0",
+    "npm-run-all": "^4.1.3"
+  },
+  "rtlcssConfig": {
+    "options": {
+        "autoRename": false,
+        "autoRenameStrict": false,
+        "blacklist":{},
+        "clean": true,
+        "greedy": false,
+        "processUrls": false,
+        "stringMap":[]
+    },
+    "plugins": [ ],
+    "map": false
+  },
+  "browserslist": [
+    "extends @wordpress/browserslist-config"
+  ],
+  "scripts": {
+    "build:style": "node-sass style.scss style.css --output-style expanded && postcss -r style.css",
+    "build:style-editor": "node-sass style-editor.scss style-editor.css --output-style expanded && postcss -r style-editor.css",
+    "build:rtl": "rtlcss style.css rtl.css",
```

Comment:
Note that `rtl.css` should only be used to overwrite the styles from the main stylesheet. Since that's not possible with rtlcss  you have change the file name to `style-rtl.css` and use `wp_style_add_data( 'twentynineteen-style', 'rtl', 'replace' );`

Reply:
Ahhh, good catch! I’ll sort that out before merging.